lib/hets-rabbitmq-wrapper.rb

```diff
@@ -7,18 +7,41 @@ module HetsRabbitMQWrapper
   def self.run
```

Comment:
Please create a class `Listener` that has all these methods as instance methods, such that the workflow is more like
```ruby
Listener.new(my_configuration).call
```
and make all methods except `initialize` and `call` private.

I suggest this because the whole wrapper will be a lot more complex than this (having different Hets tasks in mind, like parsing, proving, consistency checking, file type determination) and then we already have a reasonable entry point that loads and calls other components.